content/develop/connect/insight/release-notes/v.2.50.0.md

```diff
@@ -0,0 +1,23 @@
+---
+Title: Redis Insight v2.50.0, May 2024
+linkTitle: v2.50.0 (May 2024)
+date: 2024-05-30 00:00:00 +0000
+description: Redis Insight v2.50
+weight: 1
+
+---
+## 2.50 (May 2024)
+This is the General Availability (GA) release of Redis Insight 2.50.
+
+### Highlights
+- Optimize your Redis learning journey by signing in to your Redis Cloud account directly from Redis Insight. Easily provision a free Cloud database and level up your Redis expertise with our interactive tutorial. Or swiftly import databases from your Redis Cloud account to seamlessly proceed your journey in Redis Insight.
+- Explore common Redis use cases through a paired-up tutorial opened after uploading sample data for empty databases.
+- Performance and UX enhancements for JSON data structure in Browser
+
+### Details
+
+**Features and improvements**
+- [#3161](https://github.com/RedisInsight/RedisInsight/pull/3161), [#3171](https://github.com/RedisInsight/RedisInsight/pull/3171) Optimize your Redis learning journey by signing in to your Redis Cloud account directly from Redis Insight. Easily provision a free Cloud database and level up your Redis expertise with our interactive tutorial. Or swiftly import databases from your Redis Cloud account to seamlessly proceed your journey in Redis Insight. Integration with your Redis Cloud account is currently available only in desktop Redis Insight.
+- [#3402](https://github.com/RedisInsight/RedisInsight/pull/3402) Explore common Redis use cases through a paired-up tutorial opened after uploading sample data for empty databases.
+- [#3251](https://github.com/RedisInsight/RedisInsight/pull/3251) UX enhancements for the JSON data structure to prevent collapsing the entire structure when updating a JSON value. Along with performance optimizations for loading JSONs containing numerous objects.
```

Comment:
```suggestion
- [#3251](https://github.com/RedisInsight/RedisInsight/pull/3251) UX enhancements for the JSON data structure to prevent collapsing the entire structure when updating a JSON value. Includes performance optimizations for loading JSON documents containing numerous objects.
```